data/human/deep missions.txt

```diff
@@ -2431,3 +2431,340 @@ event "battle in arneb end"
+			choice
+				`	(Leave Grimssalr with Thorleif.)`
+					goto grimfollow
+			label grimfollow
+			`	You follow Thorleif up a mostly overgrown trail for about 30 minutes before finally stepping out into a wide open plain. The wind is cool and refreshing and the grass blows with a steadiness that is somehow reassuring. There is something special, timeless, about this sudden meadow.`
+			`	You notice over two dozen columns of lain stone arranged in a circle around a much more massive and impressive structure. The lain stone columns must be cairns of various nearby families, but the massive structure in the center is something completely different. The structure is a wall, cut from some sort of metal you don't recognize and filled with letters and images that are unfamiliar to you.`
+			`	"That is an Elf-Stone. It is why we built our Family-Cairns here. Such stones can be found on most worlds in The Deep. Indestructible, and left by the Alfar for some unknowable reason. Some, like this one, have been mostly forgotten. Others though, are important places of celebration and even worship.`
+			`	He points at a smallish cairn next to the Elf-Stone. "This, though, is what we're here for."`
+			choice
+				`	(You eye the worn and weather beaten cairn with a faded name plate reading, 'Hlewagast' carved at its base.)`
+					goto thorcarn
+			label thorcarn
+			`	Thorleif approaches the cairn.`
+			`	Thorleif holds up the stone of his father in his palm and begins to read the runes carved upon it.`
+			`	"The day my mother gave me this stone and said she was proud of me."`
+			`	"The day my son Thorleif was born."`
+			`	"The day I struck the mother lode on Helheim."`
+			`	"The day I met <first> and he helped me find my family again."`
+			`	"The day I saw my son again, even though he would not see me."`
+			`	Thorleif sniffs, and takes a moment to write something down he saw on the stone, and then walks forward and places it upon the cairn, at home now with hundreds of other stones each with their own handful of memories.`
+			`	"He had not so many good memories, I think," Thorleif said quietly. "I realize now, that mine too are quite few. Perhaps that, if nothing else, is a lesson he can teach me. For that, I thank you, dad."`
+			`	With that Thorleif turns from the cairn with blurry eyes. "It is done. Let us head back."`
+			choice
+				`	(Follow Thorleif back to the village.)`	
+			`	The two of you trudge back to the small hamlet of Grimssalr, with its people happy at having guests and long lost relatives, and to Ragnhild, happy to spend more time with Thorlief, where you feast and talk and laugh and remember.`
+			choice
+				`	(You enjoy yourself well into the night before finally succumbing to sleep.)`
+			`	In the morning the three of you catch the hovercraft back to the spaceport.`
```

Comment:
Yup, this is another spot where we'd need to get rid of the day passing before we return to the ship. This one does have a pretty easy solution though, so much so that I'll suggest it myself.
```suggestion
			`	You enjoy yourself well into the evening before Thorleif approaches you. It's time to head back.`
```
Thoughts? We can play around with this if you want.